kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -61,7 +64,11 @@ class SchemaUtilsSuite extends AnyFunSuite {
       .add("b", INTEGER)
       .add("dupColName", StringType.STRING)
     expectFailure("dupColName") {
-      validateSchema(schema, false /* isColumnMappingEnabled */ )
+      validateSchema(
```

Comment:
instead of doing this elsewhere can we add a test helper method?

Reply:
then these can each be args with default values so (a) booleans are named (b) don't need to repeat them for every call